### PR TITLE
Product add/remove image operations

### PR DESCRIPTION
**Pull request by tsilva**
This was useful for quickly adding the same image to all products to easily test images in the product listings.

### CODE REVIEW

**Comment by joamag** [comment]
@tsilva please make a unit test for this feature ... :)

**Comment by tsilva** [comment]
@joamag Added test. However, unless I set a BASE_URL in the config variables, the test fails because it tries to resolve the media url on retrieval and base url is set to None. This is triggered by this method in Media:

```python
    @classmethod
    def _get_url(cls, id):
        app = appier.get_app()
        return app.url_for("media_api.data", id = id, absolute = True)
```

File: src/budy/test/product.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        self.app.unload()
+        adapter = appier.get_adapter()
+        adapter.drop_db()
+
+    def test_add_remove_images(self):
+        media_1 = budy.Media()
+        media_1.save(validate = False)
+
+        media_2 = budy.Media()
+        media_2.save(validate = False)
```
Comment (joamag) [inline src/budy/test/product.py]: must validate data

File: src/budy/test/product.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    def setUp(self):
+        self.app = budy.BudyApp(level = logging.ERROR)
+
+    def tearDown(self):
+        self.app.unload()
+        adapter = appier.get_adapter()
+        adapter.drop_db()
+
+    def test_add_remove_images(self):
+        file = appier.typesf.ImageFile(
```
Comment (tsilva) [inline src/budy/test/product.py]: @joamag added a mock file to make the validations work

**Pull request merged**